core/iwasm/interpreter/wasm_loader.c

```diff
@@ -3713,6 +3715,16 @@ load_from_sections(WASMModule *module, WASMSection *sections,
     handle_table = wasm_interp_get_handle_table();
 #endif
 
+#if WASM_ENABLE_DYNAMIC_PGO != 0
+    module->ent_and_br_cnts = loader_malloc(
+        sizeof(void *)
+            * (module->import_function_count + module->function_count),
```

Comment:
Should we allocate counters for the import function?

Reply:
It will be easier for fast-jit to locate the counter position. `JitFrame->cur_wasm_func_idx`, which comes from *https://github.com/bytecodealliance/wasm-micro-runtime/blob/main/core/iwasm/fast-jit/jit_compiler.c#L191*, is the major reason.